### DISCUSSION TITLE
为什么导入T_State失败啊

### DISCUSSION THREAD

**Discussion by lyk1205** [nonebot/discussions#51] [问答]
File "/usr/local/lib/python3.8/dist-packages/nonebot/handler.py", line 14, in <module>
    from nonebot.typing import T_State, T_Handler
ImportError: cannot import name 'T_State' from 'nonebot.typing' (/usr/local/lib/python3.8/dist-packages/nonebot/typing.py)

**Comment by yanyongyu** [comment]
你是不是安装了nb1

**Reply by lyk1205** [reply]
我装了个插件就那样了

**Reply by yanyongyu** [reply]
卸载nb1和nb2再重新安装nb2

装的哪个插件

**Reply by lyk1205** [reply]
孤寡机器人，，，

**Reply by yanyongyu** [reply]
他依赖写错了

**Reply by lyk1205** [reply]
我刚也去看了下，他貌似依赖写成nonebot1了，然后给我安装了1
